### PR TITLE
Add `help` target to Makefile for coolness

### PR DESCRIPTION
**Pull request by Callisto13**
Total nice to have, close if not interested 😄 . We have similar things in pitch-black Makefiles, gives a chill dev experience:

I have made up missing comments (probably nonsense) and put things in what seemed to me like logical groupings 🤷‍♀️  but obvs can arrange how you like.

Screenshot 2021-09-08 at 09 32 06

### CODE REVIEW

**Comment by jpellizzari** [comment]
@Callisto13 Sorry we missed this! I will pull this down and poke it today and give you a review. It looks great at first glance!

**Review by jpellizzari — APPROVED** [review]
Looks excellent, thanks @Callisto13

**Comment by Callisto13** [comment]
👍  I've resolved the merge conflicts, hopefully I have not accidentally removed anything which should be there. LMK if it all looks fine and I will merge

File: Makefile [hunk 1]
```diff
@@ -26,87 +26,114 @@ endif
...
 
-# Run tests
-unit-tests: dependencies cmd/wego/ui/run/dist/index.html
+##@ Test
+unit-tests: dependencies cmd/wego/ui/run/dist/index.html ## Run unit tests
 	# To avoid downloading depencencies every time use `SKIP_FETCH_TOOLS=1 unit-tests`
 	KUBEBUILDER_ASSETS=$(KUBEBUILDER_ASSETS) CGO_ENABLED=0 go test -v -tags unittest ./...
 
-debug:
+fakes: ## Generate testing fakss
```
Comment (jrryjcksn) [inline Makefile:36]: Minor: fakss => fakes

**Review by jrryjcksn — APPROVED** [review]
Looks very cool! Just saw one typo

File: Makefile (hunk 1 above)
Comment (Callisto13) [reply Makefile:36]:
```suggestion
fakes: ## Generate testing fakes
```

File: Makefile [hunk 2]
```diff
@@ -26,87 +26,114 @@ endif
...
 .PHONY: bin
 
-all: wego
+all: wego ## Install dependencies and build WeGo binary
 
-# Run tests
-unit-tests: dependencies cmd/wego/ui/run/dist/index.html
+##@ Test
+unit-tests: dependencies cmd/wego/ui/run/dist/index.html ## Run unit tests
 	# To avoid downloading depencencies every time use `SKIP_FETCH_TOOLS=1 unit-tests`
```
Comment (Callisto13) [inline Makefile:33]:
```suggestion
	# To avoid downloading dependencies every time use `SKIP_FETCH_TOOLS=1 unit-tests`
```
while i am here

**Comment by jpellizzari** [comment]
@Callisto13 I will merge this once these tests pass. Thanks for your contribution!

**Pull request merged**